comms/src/backoff.rs

```diff
@@ -53,7 +53,7 @@ impl Default for ExponentialBackoff {
 
 impl Backoff for ExponentialBackoff {
     fn calculate_backoff(&self, attempts: usize) -> Duration {
-        if attempts == 0 {
+        if attempts <= 1 {
```

Comment:
Why this change?

Reply:
The connection manager counts the first attempt as 1 which would add a 2 second delay instead of starting immediately.

Reply:
The old connection manager (the one currently being used by most base node tests) starts at 0, making the first 2 attempts start immediately. Shouldn't be a problem, besides I'll be working on switching everything over to the new comms next 🎉

Reply:
The nice thing about the new comms builder is that it allows you to specify the backoff - so for tests a `ConstantBackoff::new(Duration::from_secs(0))` might be more appropriate.